docs/dockerfile_to_build_custom_logstash_image.md

```diff
@@ -0,0 +1,72 @@
+# Dockerfile to build a custom Logstash image
+
+To build an image that is not available in the [official Docker repository tags](https://hub.docker.com/r/opensearchproject/logstash-oss-with-opensearch-output-plugin/tags), or to add specific plugins to your image, you can create a custom Dockerfile.
+
+The process varies depending on whether you want to build an image with `Logstash 8.x` versions or `Logstash 7.x` versions
+
+## Logstash 8.x
+
+Create this `Dockerfile` to build an image with `Opensearch 2.0.2` for **`Logstash 8.x`**
```

Comment:
Unlike Elasticsearch, correct spelling is OpenSearch, and no need to quote since that's note code.

"Create a `Dockerfile` to build an image with OpenSearch 2.0.2 and Logstash 8.x."

Reply:
Oh, yes. Done.